webanno/webanno
# #1197 - codebook annotation support

**What's in the PR**
* Hierarchical organization of codebooks
* Replaced old Codebook Editor with Codebook Tree Editor 

**How to test manually**
1) Purge old DB and create a new and empty DB "webanno"
2) Create new Project with Codebooks and select parents 
3) Work on the Project in annotation mode. See and test new editor

**Automatic testing**

**Documentation**
